cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -635,7 +637,7 @@ private static String getAccessManagerName() {
 
     @Override
     protected void service(HttpServletRequest req, HttpServletResponse resp)
-            throws ServletException, IOException {
+            throws IOException {
```

Comment:
Isn't the `throws ServletException` part of the interface definition?

Reply:
Yes.  Sub-classes are allowed to narrow the list.  I put it back.